### ISSUE TITLE
Update operations relay client implementation

### ISSUE THREAD

**Issue by jj1980a** [FastLane-Labs/atlas-sdk#1]
The current `src/operationRelay.ts` implementation is outdated.
Update it with the latest changes  to the [atlas-operations-relay](https://github.com/FastLane-Labs/atlas-operations-relay).

A good starting point could be pasting the [REST API specs](https://github.com/FastLane-Labs/atlas-operations-relay/blob/main/specs/rest-api.yaml) into https://editor-next.swagger.io/, and generate a `client` for `typescript-fetch`. Then adjust to our specific needs. Reminder that the SDK won't use the solvers' API endpoints, we can remove that from the generated code.

For reference, the updated Atlas lifecycle is https://github.com/FastLane-Labs/atlas-operations-relay/tree/main/specs.
To summarize the SDK calls sequence to the ops relay API:
1. SDK calls `(POST)/userOperation` to submit the userOp
2. SDK calls `(GET)/solverOperations` to retrieve solverOps for the userOp submitted in step 1
3. SDK calls `(POST)/bundleOperations` to submit a bundle (user/solvers/dApp ops)
4. SDK calls `(GET)/bundleHash` to retrieve the hash of the Atlas transaction generated by the bundle submitted in step 3
